website/addons/wiki/routes.py

```diff
@@ -120,6 +120,23 @@
             '/project/<pid>/node/<nid>/wiki/<wname>/',
         ], 'delete', views.project_wiki_delete, json_renderer),
 
+        # Change Wiki Settings | PUT
+        Rule([
+            '/project/<pid>/wiki/settings/',
```

Comment:
Ping @brianjgeiger @JeffSpies for URL review.

Only feedback I have is that other addons use `/config/` (rather than `/settings/`)